base/Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM debian:sid
 MAINTAINER Zero Cho "http://itsze.ro"
 
-RUN apt-get update
-RUN apt-get install -y scala git curl openjdk-7-jdk netcat
+RUN apt-get update && apt-get install -y \
+  curl \
+  openjdk-8-jre-headless \
```

Comment:
I'm also traveling, we'll see.
On Aug 22, 2015 2:39 PM, "Adrian Cole" <notifications@github.com> wrote:

> In base/Dockerfile
> <https://github.com/openzipkin/docker-zipkin/pull/25#discussion_r37695398>
> :
>
> > @@ -1,9 +1,13 @@
> >  FROM debian:sid
> >  MAINTAINER Zero Cho "http://itsze.ro"
> >
> > -RUN apt-get update
> > -RUN apt-get install -y scala git curl openjdk-7-jdk netcat
> > +RUN apt-get update && apt-get install -y \
> > +  curl \
> > +  openjdk-8-jre-headless \
>
> Sounds good. All we'd need to add would be netcat. Wanna raise a PR? (I'm
> traveling today)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openzipkin/docker-zipkin/pull/25/files#r37695398>.
>